scipy/signal/tests/test_filter_design.py

```diff
@@ -1189,6 +1400,24 @@ def test_analog(self):
 
         assert_equal(buttord(1, 1.2, 1, 80, analog=True)[0], 55)
 
+    def test_fs_param(self):
+        wp = [4410, 11025]
+        ws = [2205, 13230]
+        rp = 3
+        rs = 80
+        fs = 44100
+        N, Wn = buttord(wp, ws, rp, rs, False, fs=fs)
+        b, a = butter(N, Wn, 'bandpass', False, fs=fs)
+        w, h = freqz(b, a, fs=fs)
+        assert_array_less(-rp - 0.1,
```

Comment:
Do you need these tolerances (0.1) to be so large?  When I run this code, I get

```
In [81]: dB(h[np.logical_and(wp[0] <= w, w <= wp[1])]).min()
Out[81]: -3.0000001585928446

In [82]: dB(h[np.logical_or(w <= ws[0], ws[1] <= w)]).max()
Out[82]: -83.080125250056454
```
so the passband check would pass with a much smaller tolerance, and the stop band passes without a tolerance.